source/utilities.tex

```diff
@@ -15881,7 +15881,7 @@
   \tcode{is_constructible_v<T,}\br
   \tcode{Args...>} is \tcode{true} and the variable
   definition for \tcode{is_constructible}, as defined below, is known to call
-  no operation that is not trivial (~\ref{basic.types},~\ref{special}). &
+  no operation that is not trivial (\ref{basic.types},~\ref{special}). &
```

Comment:
You could make a space with a positive but non-infinite penalty. But that's going too far; we can keep the non-breaking space and remove it on a case-by-case basis.